tools/importer-rest-api-specs/components/parser/models.go

```diff
@@ -177,6 +177,34 @@ func (d *SwaggerDefinition) detailsForField(modelName string, propertyName strin
 			}
 			nestedFields[propName] = *nestedField
 		}
+		for _, inlinedModel := range value.AllOf {
```

Comment:
can we add an example Swagger/test for this?

Reply:
Done 👍